packages/backend/src/api/endpoints/UserApi.ts

```diff
@@ -57,7 +57,7 @@ type CreateUserParams = {
   totpSecret?: string;
   backupCodes?: string[];
 } & UserMetadataParams &
-  (UserPasswordHashingParams | {});
+  (UserPasswordHashingParams | object);
```

Comment:
Here (and in other instances) `{}` was changed to `object` for now since that's what the author intended. Long-term it probably makes sense to convert this to `Record<string, unknown>` via ESLint rule